Common/cpp/ReanimatedRuntime/ReanimatedHermesRuntime.cpp

```diff
@@ -0,0 +1,86 @@
+#include "ReanimatedHermesRuntime.h"
```

Comment:
You can move this to `#if JS_RUNTIME_HERMES` if you want.

Btw why don't we compile this file only `#if JS_RUNTIME_HERMES && HERMES_ENABLE_DEBUGGER`?

Reply:
We could initialize the runtime differently, when `HERMES_ENABLE_DEBUGGER` is not set, but I think it looks cleaner when it is consistent. Though I could wrap the adapter in `#if HERMES_ENABLE_DEBUGGER` to exclude it in non-debug builds